api/internal/plugins/execplugin/execplugin_test.go

```diff
@@ -53,48 +52,45 @@ s/$BAR/bar baz/g
 	if err != nil {
 		t.Fatalf("unexpected err: %v", err)
 	}
-	p := NewExecPlugin(pluginPath)
-	// Not checking to see if the plugin is executable,
-	// because this test does not run it.
-	// This tests only covers sending configuration
-	// to the plugin wrapper object and confirming
-	// that it's properly prepared for execution.
+	for _, pPath := range pluginPath {
```

Comment:
Per my comment in the other file, I don't think `loader.AbsolutePluginPath` should return multiple values, so this test shouldn't require changes.